app/views/clients/rollup/_disabilities.haml

```diff
@@ -30,7 +30,11 @@
             %td= project&.name(current_user)
             %td.date-cell= disability.enrollment.EntryDate
             %td= disability.InformationDate
-            %td= HUD.disability_type disability.DisabilityType
+            %td
+              - if disability.DisabilityType == 8 && !can_view_hiv_status?
+                %i.light redacted
+              - else
+                = HUD.disability_type disability.DisabilityType
```

Comment:
Angela found this during QA–– figure it makes sense to throw into release-38 with the rest of the confidentiality updates.


<img width="712" alt="Screen Shot 2022-06-22 at 5 05 38 PM" src="https://user-images.githubusercontent.com/5333982/175064746-a63e4b9c-8b40-4a25-b77c-e0077226ae74.png">